### PR TITLE
Avatar & badge

### PR DESCRIPTION
**Pull request by Creator-singh**
Completed Avatar and Badge Component and also added it to documentation.

### CODE REVIEW

File: components/badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    justify-content: space-around;
+    padding: 2rem;
+    text-align: center;
+    border: 1px solid lightgrey;
+    padding: 1.5rem;
+    box-shadow: 3px 3px 3px 3px lightgrey;
+}
+
+.user{
+    position: relative;
```
Comment (anupkgurung) [inline components/badge/badge.css:21]: what does user signify ?

File: components/badge/badge.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+<body>
+    <section id="badge">
+        <h3>Badge</h3>
+        <p>
+            A badge is a visual indicator which can contain numeric values or just a simple indicator usually used for
+            notifications, user status or on cards.
+        </p>
+
+        <div class="badge-example">
```
Comment (anupkgurung) [inline components/badge/badge.html:26]: extra space can be removed.

File: pages/documentation.html [hunk 3]
```diff
@@ -57,27 +58,124 @@
...
+        
+                <div class="hidden">
+                    I am hidden
+                </div>
+        
+                <div class="snippet">
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%28211%2C211%2C211%2C1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=4x&wm=false&code=%253Cdiv%2520class%253D%2522user%2522%253E%250A%2509%253Cimg%2520class%253D%2522rounded-img%2520avatar-md%2522%2520src%253D%2522https%253A%252F%252Fi.pravatar.cc%252F400%2522%2520alt%253D%2522avatar-md%2522%253E%250A%2509%253Cspan%2520class%253D%2522active%2522%253E%253C%252Fspan%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522fas%2520fa-envelope%2520badge-icon%2522%253E%253Cspan%2520class%253D%2522badge%2522%253E2%253C%252Fspan%253E%253C%252Fi%253E%250A%253C%252Fdiv%253E"
+                        style="width: 100%; height: 299px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
```
Comment (anupkgurung) [inline pages/documentation.html:156]: you can add a class where you can define width and height for iframe to be consistent all over the application.

File: utilities/reset.css [hunk 4]
```diff
@@ -31,6 +28,37 @@ ul {
...
+	font-size: 1.8rem;
+	line-height: 2.4rem;
+}
+
+h5 {
+	font-size: 1rem;
+	line-height: 1.6rem;
+}
+
+h6 {
```
Comment (anupkgurung) [inline utilities/reset.css:57]: before commting the code you should do a proper indentation. To much white space.

**Review by anupkgurung — CHANGES_REQUESTED** [review]
Please use proper indentation before pushing the code.

File: components/avatar/avatar.css [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+code{
+    background-color: lightgray;
+}
+
+.avatar-example{
+    text-align: center;
+    border: 1px solid lightgrey;
+    padding: 1.5rem;
+    box-shadow: 3px 3px 3px 3px lightgrey;
```
Comment (rohanmathur91) [inline components/avatar/avatar.css]: Use variables for colors.

File: components/avatar/avatar.css [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+.hidden{
+    visibility: hidden;
+}
+
+.snippet{
+    border: 1px solid lightgrey;
+    padding: 1.5rem;
+    box-shadow: 3px 3px 3px 3px lightgrey;
+}
```
Comment (rohanmathur91) [inline components/avatar/avatar.css:63]: Same as above.

**Review by rohanmathur91 — COMMENTED** [review]
Looks good to me.

**Review by rohanmathur91 — APPROVED** [review]

File: components/badge/badge.css (hunk 1 above)
Comment (Creator-singh) [reply components/badge/badge.css:21]: It was the image user for activity status on user. Changed it to user image.

File: pages/documentation.html (hunk 3 above)
Comment (Creator-singh) [reply pages/documentation.html:156]: yes, but it changes based on the length of code in the snippet. So didn't added.

File: utilities/reset.css (hunk 4 above)
Comment (Creator-singh) [reply utilities/reset.css:57]: okay

**Review by anupkgurung — APPROVED** [review]
looks good !!

**Pull request merged**